install/kubernetes/helm/istio/charts/tracing/templates/service-jaeger.yaml

```diff
@@ -22,7 +26,7 @@ items:
         targetPort: {{ .Values.service.uiPort }}
     selector:
       app: jaeger
-    type: LoadBalancer
+    type: {{ .Values.service.type }}
```

Comment:
This needs to be defined as a different type to the collector service - so can you create a different property in the values.yaml (e.g. .Values.service.uiType) which should be set to `LoadBalancer`?

Reply:
in fact, much better. I just made the change. Thanks